tests/ci/clean_up.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
The following syntax is probably prefered: `#!/usr/bin/env bash`

Reply:
I actually prefer the other syntax, so I know that binary is running (not some random `bash` at the `PATH`), and `/bin/` is the common place for `bash` to live in any modern Linux.

I don't mind to change it, but maybe we can agree on some style, so everyone writes the same way. I like this [styleguide](https://google.github.io/styleguide/shellguide.html).
Anyway, I don't think we're going to write a lot of shellscripts... do we?

Reply:
You are probably right. I do not have a strong opinion on this. approving the PR.